common/lib/xmodule/xmodule/js/src/video/09_bumper.js

```diff
@@ -79,6 +81,7 @@ define(
             var events = ['ended', 'skip', 'error'].join(' ');
             this.element.off(events, this.showMainVideo);
             if (_.isFunction(this.state.videoPlayer.destroy)) {
+                this.state.videoSaveStatePlugin.onSkip(this.doNotShowAgain);
```

Comment:
@polesye  why? This is related to the question about place we should fire saveSate.